diagrams/programming/framework.py

```diff
@@ -71,6 +71,8 @@ class Svelte(_Framework):
 class Vue(_Framework):
     _icon = "vue.png"
 
+class Quarkus(_Framework):
```

Comment:
Also shouldn’t classes be ordered alphabetically?

Reply:
I'm fixing this :zap: